hw_4/src/app/components/WelcomeModal.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+export default class WelcomeModal extends Component {
+    constructor(props) {
+		super(props);
+		this.state = {
+			display: true
```

Comment:
```suggestion
			display: false
```

и ниже надо добавить:
```
componentDidMount() {
    this.setState({ display: true });
}
```